application-analytics-ui/src/main/resources/Analytics/Code/Macros/MostViewedPages.xml

```diff
@@ -241,8 +241,12 @@
     </property>
     <property>
       <code>{{include reference='Analytics.Code.Macros.VelocityMacros'/}}
+{{include reference="Licenses.Code.VelocityMacros"/}}
 
 {{velocity}}
+#if ($services.licensing.licensor.hasLicensureForEntity(
+    $services.model.createDocumentReference('', ['Analytics', 'Code'], 'ConfigurationClass')))
+
```

Comment:
There is an extra newline displaye dabove the mostViewedPages macro, and it might come from here